scripts/ci/run-test-make-install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+${TESTCMD} --label=make-install --timeout=30m make -w install
+mkdir -p ${MONO_REPO_ROOT}/tmp/
+echo "class H{static void Main(){System.Console.WriteLine(\"ok\");}}" > ${MONO_REPO_ROOT}/tmp/hello.cs
+export LD_LIBRARY_PATH=${MONO_REPO_ROOT}/tmp/monoprefix/lib:${LD_LIBRARY_PATH}
```

Comment:
Instead of just setting LD_LIBRARY_PATH we should do:

```bash
MONO_PREFIX=${MONO_REPO_ROOT}/tmp/monoprefix
export DYLD_FALLBACK_LIBRARY_PATH=$MONO_PREFIX/lib:$DYLD_LIBRARY_FALLBACK_PATH
export LD_LIBRARY_PATH=$MONO_PREFIX/lib:$LD_LIBRARY_PATH
export PATH=$MONO_PREFIX/bin:$PATH
```

and then use `mcs`, `csc` etc.